apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -96,6 +96,10 @@ interface AuthenticationFlowPropsInterface extends TestableComponentInterface {
      * Update authentication steps.
      */
     updateSteps: (add: boolean) => void;
+    /**
+     * Callback to update the button disable state change.
+     */
+    onAuthenticationSequenceChange: (buttonStateDisabled: boolean) => void;
```

Comment:
Shall we rename this to `isDisabled`? WDYT?
```suggestion
    onAuthenticationSequenceChange: (isDisabled: boolean) => void;
```

Reply:
fixed in 7402005b3cafad3dd06a3f785122cb2d30e11cf4